snapshotter/demux/proxy/address/resolver.go

```diff
@@ -25,6 +25,9 @@ type Response struct {
 	//
 	// Reference: https://pkg.go.dev/net#SplitHostPort
 	Address string `json:"address"`
```

Comment:
Can we split `SnapshotterPort` out of `Address` now that we have a use-case for multiple guest services using the same host addresss?

Reply:
Sure thing, that'll make a Response look like:

```
Response{
    Address: /path/to/uds.sock,
    SnapshotterPort: 1234,
    MetricsPort: 5678,
}
```